packages/story-editor/src/components/mediaRecording/footer.js

```diff
@@ -276,6 +282,37 @@ function Footer({ captureImage, videoRef }) {
     speak,
   ]);
 
+  const {
+    actions: { uploadFile },
+  } = useUploader();
+
+  const { updateStory } = useStory(({ actions: { updateStory } }) => ({
+    updateStory,
+  }));
+  const { setHighlights } = useHighlights(({ setHighlights }) => ({
+    setHighlights,
+  }));
+
+  const onAudioInsert = async () => {
```

Comment:
I suppose this does _not_ need `useCallback` because it's passed directly to a button element and not another React component?

Reply:
Yep, this just goes to the button via styled Button, can't see `useCallback` benefits here.